wolfssl/wolfcrypt/settings.h

```diff
@@ -211,6 +211,42 @@
     #define NO_FILESYSTEM
 #endif
 
+#if defined(WOLFSSL_CONTIKI)
+    #include <contiki.h>
+    #define UIP
+    #define NO_WRITEV
+    #define SINGLE_THREADED
+    #define WOLFSSL_USER_IO
+    #define NO_FILESYSTEM
+    #define CUSTOM_RAND_TYPE uint16_t
+    #define CUSTOM_RAND_GENERATE random_rand
+    static inline unsigned int LowResTimer(void)
+    {
+        return clock_seconds();
+    }
+
+    static inline void* XREALLOC(void *p, size_t n, void* heap, int type)
```

Comment:
Is there a reason to not use the existing types.h maps for these which are here: https://github.com/wolfSSL/wolfssl/blob/master/wolfssl/wolfcrypt/types.h#L233

Just define `NO_WOLFSSL_MEMORY` to use it.